docs/app/Examples/elements/Icon/IconSet/IconSetExamples.js

```diff
@@ -1,16 +1,21 @@
+import _ from 'lodash'
 import React, { Component } from 'react'
 import ComponentExample from 'docs/app/Components/ComponentDoc/ComponentExample'
 import ExampleSection from 'docs/app/Components/ComponentDoc/ExampleSection'
+import { iconCats as categories } from 'src/utils/semanticutils'
 
 export default class IconSetExamples extends Component {
   render() {
     return (
       <ExampleSection title='Icon Set'>
-        <ComponentExample
-          title='Web Content'
-          description='Icons can represent types of content found on websites'
-          examplePath='elements/Icon/IconSet/IconWebContentExample'
-        />
+        {_.map(categories, category => (
+          <ComponentExample
+            title={category.name}
+            description='Icons can represent types of content found on websites'
+            examplePath='elements/Icon/IconSet/IconCategoryExample'
+            category={category}
+          />
+        ))}
       </ExampleSection>
```

Comment:
I'm not sure this is desirable here, as the each example now being dynamically generated has the side affect of render the example's source view, unintelligible for the user!

Was hoping to avoid having to manually create an example per icon category, but I believe that's what you're going to tell me!